mod_deploy/controllers.py

```diff
@@ -72,6 +81,12 @@ def is_valid_signature(x_hub_signature, data, private_key):
 @mod_deploy.route('/deploy', methods=['GET', 'POST'])
 @request_from_github()
 def deploy():
+    """
+    Deploy the GitHub request to the test platform.
+    
+    :request, abort_code, g: library imported by flask
```

Comment:
This is not the correct and wanted type of annotation.

Only arguments of the function (`:param` and the type of that parm `:type`) should be added. Please take a look at the example I linked yesterday.

Reply:
No need to add everything?
only the important one?

Reply:
Deploy has no arguments, so besides the description nothing is needed here.